docs/make.jl

```diff
@@ -90,7 +90,8 @@ pages = [
     "Ecosystem Integration" => [
         "LightGraphs" => "examples/schoolyard.md",
         "DifferentialEquations" => "examples/diffeq.md",
-        "BlackBoxOptim" => "examples/optim.md"
+        "BlackBoxOptim" => "examples/optim.md",
+        "Measurements" => "examples/measurements.md",

```

Comment:
I wonder, should we put the ending `.jl` in all page headers?

Reply:
Perhaps. I guess it's essentially the convention for repos now. So will do this when I merge.